File: IT/Database/Esercizi/Python/esercizio4.py
# file esercizio4.py

from __future__ import print_function
import sqlite3

conn = sqlite3.connect("students.db")
cursor = conn.cursor()

try:
	cursor.execute("DELETE FROM students")
	conn.commit()
	print("Tabella degli studenti svuotata.")

except sqlite3.OperationalError as e:
	print(e)

File: IT/Database/Esercizi/C++/esercizio4.cpp
/* file esercizio4.cpp */

#include <iostream>
#include <sqlite3.h>

using std::cout;
using std::endl;

int main()
{
	sqlite3 *db;
	sqlite3_open("students.db", &db);

	int rc = sqlite3_exec(db, "DELETE FROM students", NULL, NULL, NULL);

	if (rc != SQLITE_ERROR) {
		cout << "Tabella degli studenti svuotata." << endl;
	}
	else {
		cout << sqlite3_errmsg(db) << endl;
	}

	sqlite3_close(db);
}

